doc/nrf/test_and_optimize/optimizing/power_general.rst

```diff
@@ -86,3 +91,103 @@ For example, for the nRF9160 SiP, see the `Electrical specification of nRF9160`_
    An I/O pin with a 10 kΩ pull-up resistor that is set to ``GND`` will result in a current consumption of 300 µA at 3V.
+
+Disable unused pins and peripherals
+===================================
+
+.. disable_unused_pins_start
+
+Some of the pins and peripherals are enabled by default for some boards.
+Depending on the peripheral or the pin type, they can increase the device power consumption to a different extent.
+If the application does not use them, make sure they are disabled.
+
+To disable a particular peripheral, set its state in the board's :file:`dts` overlay to ``disabled``.
+For example, for **ADC**:
+
+.. code-block:: devicetree
+
+    &adc {
+        status = "disabled";
+    };
+
+.. disable_unused_pins_end
+
+Enable Device Power Management module
+=====================================
+
+.. enable_device_pm_start
+
+The Device Power Management module provides an interface that the device drivers use to be informed about entering the suspend state or resuming from the suspend state.
+This allows the device drivers to do any necessary power management operations, such as turning off device clocks and peripherals, which lowers the power consumption.
+
+To enable suspending peripherals when CPU goes to sleep, set the :kconfig:option:`CONFIG_PM_DEVICE` Kconfig option to ``y``.
+
+.. enable_device_pm_end
+
+Put the external flash into sleep mode in inactivity periods
+============================================================
+
+When the CPU goes to sleep, some of the peripherals are suspended by their drivers.
+However, the driver is not always able to know the application behavior and optimally handle the peripheral state.
+
+One of such cases is the external flash usage by the applications.
+It is typically used very rarely and only for the Device Firmware Upgrade purposes.
+For this reason, you might want to suspend the external flash for majority of time and have it resumed to the active state only if needed.
+The Device Firmware Upgrade case is properly handled in the |NCS|, but for other proprietary use cases, you should handle state changes in your own implementation.
+
+For example, to control the QSPI NOR external flash, you can use the following implementation:
+
+.. code-block:: C++
+
+    #include <zephyr/pm/device.h>
+
+    const auto * qspi_dev = DEVICE_DT_GET(DT_INST(0, nordic_qspi_nor));
+    if (device_is_ready(qspi_dev))
+    {
+        // Put the peripheral into suspended state.
+        pm_device_action_run(qspi_dev, PM_DEVICE_ACTION_SUSPEND);
+
+        // Resume the peripheral from the suspended state.
+        pm_device_action_run(qspi_dev, PM_DEVICE_ACTION_RESUME);
+    }
+
+Configure radio transmitter power
+=================================
+
+.. radio_power_start
+
+The radio transmitter power (radio TX power) has a significant impact on the device power consumption.
+The higher the transmitting power, the greater the wireless communication range, which leads to higher power consumption.
```

Comment:
```suggestion
The higher the transmitter power, the greater the wireless communication range, but it also leads to higher power consumption.
```
This is the only line with "transmitting". I'd recommend using the same word for the whole section, even though both words are grammatically correct.